tasks/main.yml

```diff
@@ -10,3 +10,41 @@
   package:
     name: cockpit-session-recording
   when: install_session_player
+
+- name: configure basic sssd
+  template:
+    src: 'etc/sssd/conf.d/sssd-session-recording.conf.j2'
+    dest: /etc/sssd/conf.d/sssd-session-recording.conf
+    owner: root
+    group: root
+    mode: 0600
+
+- name: configure sssd session recording config
+  template:
+    src: 'etc/sssd/sssd.conf.j2'
+    dest: /etc/sssd/sssd.conf
+    owner: root
+    group: root
+    mode: 0600
+
+- name: configure sssd session recording config
+  template:
+    src: 'etc/tlog/tlog-rec-session.conf.j2'
+    dest: /etc/tlog/tlog-rec-session.conf
+    owner: root
+    group: root
+    mode: 0600
+
+# for future purpose. in template use _UID or keep programname if it works
+- name: getent for tlog
+  getent:
+    database: passwd
+    key: tlog
+
+- name: configure rsyslog to elastic logging
```

Comment:
Let's add a `when` condition to this that only runs this task if a variable like `session_recording_output: elasticsearch` is specified.